config/seata-config-core/src/main/java/io/seata/config/FileConfiguration.java

```diff
@@ -306,7 +308,9 @@ public FileListener(String dataId, ConfigurationChangeListener listener) {
 
         @Override
         public void onChangeEvent(ConfigurationChangeEvent event) {
-            while (true) {
+            ScheduledExecutorService onChangeEventExecutor =
```

Comment:
I think ScheduledExecutorService would be instantiate more time if I listen different config.

Reply:
PTAL @l81893521